packages/cube/pipeline/check-observations.ttl

```diff
@@ -45,7 +45,8 @@ _:sortChunkSize a p:Variable ;
         [ n3:parse () ]
         _:sortBySubject
         [ splitDataset:bySubject () ]
-        _:batch
+        [ base:batch ("batchSize"^^p:VariableName) ]
+        [ base:flatten () ]
```

Comment:
`flatten` was part of an experiment?

Reply:
here `flatten` is needed because chunks produces by `base:batch` are arrays of arrays (due to the preceding split by subject)

Reply:
and in the previous `_:batch` implementation we were essentially doing the batching (`chunkObjectsBySize`) and the flattening (`chunkBySize`)

Reply:
Got it. Thanks